core/src/operator.rs

```diff
@@ -598,22 +606,17 @@ where
 
         txs_to_be_sent.push(slash_or_take_tx_handler.tx.raw_hex());
 
-        tracing::debug!(
-            "Found txs to be sent with slash_or_take_tx: {:?}",
-            txs_to_be_sent
-        );
-
-        let move_tx_handler = transaction_builder::create_move_tx(
+        let move_tx_handler = transaction_builder::create_move_tx_handler(
```

Comment:
why not use create_move_tx here

Reply:
3603f7331a703f8f5aea7770fe63cfae18cdb778